File: baekjoon/python/1822.py
import sys; input = sys.stdin.readline

AN, BN = map(int, input().split())
result = {i: 0 for i in input().split()}

for i in input().split():
    if i in result: result.pop(i)

print(len(result))
print(*sorted(map(int, result.keys())))

File: baekjoon/javascript/1822.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];
rl.on("line", line => {
  input.push(line);
}).on("close", () => {
  const [N, M] = input[0].split(" ");
  const A = new Set(input[1].split(" "));
  const B = new Set(input[2].split(" "));
  for (let b of B) if (A.has(b)) A.delete(b);
  const result = Array.from(A.values()).sort((a, b) => a - b);
  console.log(result.length);
  console.log(...result);
  process.exit();
});
